database/src/schema/account.rs

```diff
@@ -110,6 +110,21 @@ impl Account {
         .map(|_| ())
     }
 
+    // Delete an account
+    //
+    // Do not use this outside of testing. When we support account deletion,
+    // there will be a process involved to ensure we're complying with
+    // international privacy laws.
+    pub async fn delete<'e, E: sqlx::Executor<'e, Database = sqlx::Postgres>>(
```

Comment:
Shouldn't we still unit test this while we are at it?

Reply:
I mean, it's being included in the test surface, and it's not meant to be used in production, so I didn't see a need to unit test functionality that is only meant to be called within a test.

Unfortunately because of the same `#[cfg(test)]` limitation I described in the other comment, I can't make this method only available in tests. That's what I would prefer to do.

Reply:
> Do not use this outside of testing.

Yes, as you said in your own comment. Sry.

We can do the feature thing I described above to work around that issue.

Reply:
Going to wrap this one in the feature flag, but the others will be unit tested.